[docs] update README according to new project (#1)
ostis-ai/scl-machine: README.md

# SCL-machine
Inference module for [OSTIS project](https://github.com/ostis-ai/ostis-project)

# Installation
```sh
git clone https://github.com/ostis-apps/scl-machine.git
cd scl-machine/
git submodule update --init --recursive
```

## Build scp-machine

> Before build scp-machine make sure that sc-machine is assembled. Sc-machine and scp-machine must be allocated in the same folder.

Reviewer: ```suggestion
Before build scp-machine make sure that sc-machine had been built. Sc-machine and scp-machine must be allocated in the same folder.
```
